docs/pages/choose-an-edition/teleport-cloud/faq.mdx

```diff
@@ -138,40 +138,49 @@ The ability to download recordings for offline viewing will be available in a fu
 
 ### Will Teleport be automatically upgraded with each release?
 
-Teleport Cloud follows the Teleport [Upcoming
-Releases](../../upcoming-releases.mdx) timelines.
+Teleport Team and Teleport Enterprise Cloud are upgraded after each Teleport release
```

Comment:
We don't necessarily upgrade to each release. I think we should generalize this statement a bit and just say something along the lines of:

- we keep your auth and proxy patched and up to date
- patches and minors can be applied as soon as the day after a release
- major upgrades are not applied immediately (1 month delay)